app/models/proxy.rb

```diff
@@ -51,23 +51,23 @@ class Proxy < ApplicationRecord
 
   validates :credentials_location, inclusion: { in: %w[headers query authorization], allow_nil: false }
 
-  validates :error_status_no_match, :error_status_auth_missing, :error_status_auth_failed,
+  validates :error_status_no_match, :error_status_auth_missing, :error_status_auth_failed, :error_status_limits_exceeded,
                             numericality: { greater_than_or_equal_to: 200, less_than: 600 }
 
   validates :auth_app_id, :auth_app_key, :auth_user_key, :secret_token,
                       format: { with: APP_OR_USER_KEY, allow_nil: false }
 
   validates :error_headers_auth_failed,
                       :error_headers_auth_missing, :error_headers_no_match,
-                      :error_auth_failed,
-                      :error_auth_missing, :error_no_match,
+                      :error_auth_failed, :error_auth_missing, :error_no_match,
+                      :error_headers_limits_exceeded, :error_limits_exceeded,
                       format: { with: HTTP_HEADER }
 
   validates :api_test_path, length: { maximum: 8192 }
   validates :endpoint, :api_backend, :auth_app_key, :auth_app_id, :auth_user_key,
             :oidc_issuer_endpoint, :oidc_issuer_type,
             :credentials_location, :error_auth_failed, :error_auth_missing, :authentication_method,
-            :error_headers_auth_failed, :error_headers_auth_missing, :error_no_match,
+            :error_headers_auth_failed, :error_headers_auth_missing, :error_headers_limits_exceeded, :error_no_match,
```

Comment:
you will need also `error_limits_exceeded` here to make the tests pass 😄

Reply:
true.